data/human/fleets.txt

```diff
@@ -3488,3 +3488,105 @@ fleet "Human Wormhole Guards"
 		"Frigate (Mark II)"
 	variant 5
 		"Rainmaker (Mark II Old Weapons)"
+
+fleet "Derelict Republic"
+	government "Derelict (Boardable)"
+	names "civilian"
+	"cargo settings"
+		cargo 2
+		outfitters "Derelict Republic"
+	personality derelict mute marked
+	variant 18
+		"Scout (Derelict)"
+	variant 12
+		"Hauler (Derelict)"
+	variant 5
+		"Hauler II (Derelict)"
+	variant 18
+		"Berserker (Derelict)"
+	variant 15
+		"Freighter (Derelict)"
+	variant 2
+		"Star Queen (Derelict)"
+	variant 14
+		"Firebird (Derelict)"
+	variant 12
+		"Aerie (Derelict)"
+	variant 1
+		"Leviathan (Derelict)"
+	variant 20
+		"Sunder (Derelict)"
+
+fleet "Derelict Southern"
+	government "Derelict (Boardable)"
+	names "civilian"
+	"cargo settings"
+		cargo 2
+		outfitters "Derelict Southern"
+	personality derelict mute marked
+	variant 24
+		"Argosy (Derelict)"
+	variant 12
+		"Hauler (Derelict)"
+	variant 24
+		"Clipper (Derelict)"
+	variant 30
+		"Hawk (Derelict)"
+	variant 30
+		"Fury (Derelict)"
+	variant 16
+		"Bastion (Derelict)"
+	variant 12
+		"Osprey (Derelict)"
+	variant 3
+		"Firebird (Derelict)"
+	variant 15
+		"Freighter (Derelict)"
+	variant 1
+		"Falcon (Derelict)"
+	variant 24
+		"Sunder (Derelict)"
+	variant 2
+		"Hogshead (Derelict)"
+
+fleet "Derelict Syndicate"
+	government "Derelict (Boardable)"
+	names "civilian"
+	"cargo settings"
+		cargo 2
+		outfitters "Derelict Syndicate"
+	personality derelict mute marked
+	variant 15
+		"Bounder (Derelict)"
+	variant 10
+		"Hauler (Derelict)"
+	variant 25
+		"Freighter (Derelict)"
+	variant 2
+		"Bulk Freighter (Derelict)"
+	variant 10
+		"Quicksilver (Derelict)"
+	variant 3
+		"Firebird (Derelict)"
+	variant 9
+		"Splinter (Derelict)"
+	variant 3
+		"Protector (Derelict)"
+	variant 20
+		"Sunder (Derelict)"
+
+fleet "Derelict Pirate"
+	government "Derelict (Boardable)"
+	names "civilian"
```

Comment:
Pirate names for pirate fleets, perhaps.
```suggestion
	names "pirate"
```
Although the Freighter and Hauler being in this fleet wouldn't make the most sense.